frontend/src/app/api/categories/route.ts

```diff
@@ -1,23 +1,45 @@
-import { pb } from "@/lib/pocketbase";
-import { Category } from "@/lib/types";
+import { basePath } from "@/config/siteConfig";
+import { Category, Script } from "@/lib/types";
 import { NextResponse } from "next/server";
 
 export const dynamic = "force-static";
 
+const fetchCategories = async (): Promise<Category[]> => {
+  const response = await fetch(
+    `https://raw.githubusercontent.com/community-scripts/${basePath}/refs/heads/main/json/metadata.json`,
+  );
+  const data = await response.json();
+  return data.categories;
+};
+
+const fetchScripts = async (): Promise<Script[]> => {
+  const response = await fetch(
+    `https://api.github.com/repos/community-scripts/${basePath}/contents/json`,
+  );
+  const files: { download_url: string }[] = await response.json();
+  const scripts = await Promise.all(
+    files.map(async (file) => {
```

Comment:
```suggestion
    files.map(async (file): Script => {
```
Would this work instead of forcing the type below?